### ISSUE TITLE
3.1.5 installation fails on Linux

### ISSUE THREAD

**Issue by JulienFavre** [fyookball/electrum#579]
I have Ubuntu 17.10, I can run 3.1.5 properly with ./electron-cash
But when I try to install it with "sudo python3 setup.py install", I get the following error:
```
Traceback (most recent call last):
  File "setup.py", line 12, in <module>
    with open('contrib/requirements/requirements.txt') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'contrib/requirements/requirements.txt'
```
Thanks!

**Comment by hegjon** [comment]
I am getting the same error when I try to update the Fedora RPM package: https://kojipkgs.fedoraproject.org//work/tasks/5289/25415289/build.log

**Comment by anth-ny** [comment]
Getting the same error with my Dockerized build (Ubuntu 17.10).

**Comment by marceloneil** [comment]
Use the github tarball instead: https://github.com/fyookball/electrum/archive/3.1.5.tar.gz
The packaged version seems to be missing that file

**Comment by marceloneil** [comment]
The packaged version should include these files in the next release, but for now you need to use the tagged source-code tarball

**Comment by JulienFavre** [comment]
Ok, it worked.
Thanks!

**Comment by einsteinsfool** [comment]
I dislike the solution. Having not working tarball on the official page of Electron Cash and having to download it from GitHub? Is updating the package, checksums and signatures really that hard?
**Please reopen so it's fixed.**

Also for the packages on GitHub there are no checksums (for which I'll make another issue) so users can't verify the checksums. Updating the tarball is therefore critical! What happened to the idea from the main page of Electron Cash?

> Please, please, PLEASE... take the time to verify
the authenticity of these files before running them.
This protects you from hackers and malware!

**Comment by yura-pakhuchiy** [comment]
Also would be great to have link to signature near download link for package itself. Rather than forcing every user to search for signature file in repository on github.

**Comment by marceloneil** [comment]
@einsteinsfool you're right, we should repackage and sign a new package. I originally thought it would require some code changes (and therefore not be deterministic) - but that turned out not to be the case.

However, the GitHub tarball that I linked should be trusted. It is generated and provided by GitHub, so if you trust them (which this project does), the tarball can be trusted as well. It's pretty much equivalent to cloning the repository and rolling back to a specific tag.

**Comment by marceloneil** [comment]
@hegjon it should work fine with dd409ac24aafb7fd9b3e0cda8f1ffb132daaef78 or with a recent version of setuptools

**Comment by marceloneil** [comment]
This issue is resolved in 3.1.6
